### PR TITLE
adding a CHIP Online Presence

### PR DESCRIPTION
**Pull request by eren885**
fRsM
SpJZ
ZX89
CXTm
3sAb

### CODE REVIEW

File: CHIP Online/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    },
+    "service": "CHIP Online",
+    "description": {
+        "tr": "Teknoloji/spor/gündem ve insanların blog oluşturduğu, Türkiye'de popüler bir site.",
+         "en": "CHIP Online is the independent web portal of the CHIP brand."
+    },
+    "url": ["https://www.chip.com.tr/", "https://www.chip.com.tr", "www.chip.com.tr"],
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/attachments/618484091305852931/651438697916530751/VgD5xEef.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/618484091305852931/651438697916530751/VgD5xEef.png",
```
Comment (frozaum) [inline CHIP Online/dist/metadata.json]: host the images on imgur or some image hosting service

File: CHIP Online/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "author": {
+        "name": "Eren",
+        "id": "478937019758673931"
+    },
+    "service": "CHIP Online",
+    "description": {
+        "tr": "Teknoloji/spor/gündem ve insanların blog oluşturduğu, Türkiye'de popüler bir site.",
+         "en": "CHIP Online is the independent web portal of the CHIP brand."
+    },
+    "url": ["https://www.chip.com.tr/", "https://www.chip.com.tr", "www.chip.com.tr"],
```
Comment (frozaum) [inline CHIP Online/dist/metadata.json]: it isn't necessary to add the link three times, just use www.chip.com.tr

File: CHIP Online/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/attachments/618484091305852931/651438697916530751/VgD5xEef.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/618484091305852931/651438697916530751/VgD5xEef.png",
+    "color": "#1cb6e6",
+    "tags": [
+    "social"
+    ],
+    "category": "other",
+    "regExp": "(([a-z0-9]+)[.])?chip[.]([a-z0-9]+)"
```
Comment (frozaum) [inline CHIP Online/dist/metadata.json]: remove this since yeah

File: CHIP Online/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        "tr": "Teknoloji/spor/gündem ve insanların blog oluşturduğu, Türkiye'de popüler bir site.",
+         "en": "CHIP Online is the independent web portal of the CHIP brand."
+    },
+    "url": ["https://www.chip.com.tr/", "https://www.chip.com.tr", "www.chip.com.tr"],
+    "version": "1.0",
+    "logo": "https://cdn.discordapp.com/attachments/618484091305852931/651438697916530751/VgD5xEef.png",
+    "thumbnail": "https://cdn.discordapp.com/attachments/618484091305852931/651438697916530751/VgD5xEef.png",
+    "color": "#1cb6e6",
+    "tags": [
+    "social"
```
Comment (frozaum) [inline CHIP Online/dist/metadata.json]: this is a news website, please change the tags

File: CHIP Online/dist/presence.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    mediaKeys: false
+});
+
+presence.on("UpdateData", () => {
+    let presenceData = {
+        largeImageKey: "logo-chip"
+    };
+   
+    
+
```
Comment (Bas950) [inline CHIP Online/dist/presence.js]: Why so many spaces everywhere

File: CHIP Online/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    "author": {
+        "name": "Eren",
+        "id": "478937019758673931"
+    },
+    "service": "CHIP Online",
+    "description": {
+        "tr": "Teknoloji/spor/gündem ve insanların blog oluşturduğu, Türkiye'de popüler bir site.",
+         "en": "CHIP Online is the independent web portal of the CHIP brand."
+    },
+    "url": "http://www.chip.com.tr",
```
Comment (Bas950) [inline CHIP Online/dist/metadata.json]: remove http://

**Pull request merged**